internal/ctrlutils/ingress-status.go

```diff
@@ -120,11 +120,23 @@ func (s *StatusUpdater) PullConfigUpdate(
 ) {
 	cfg := statusConfig{ready: false}
 	status := statusInfo{ready: false}
+	latest := time.Unix(0, 0)
 	var wg sync.WaitGroup
 	var err error
 	for {
 		select {
 		case updateDone := <-s.kongConfig.ConfigDone:
+			s.log.V(util.DebugLevel).Info("status handler received config", "timestamp", updateDone.Timestamp,
+				"latest", latest)
```

Comment:
nit: it's very minor but technically the first pass through this will log a timestamp that at best will be meaningless, but at worst confusing for anyone reading the logs. We could wrap the timestamp in an object which could produce a "no previous configuration" string for logging purposes and then the string timestamp after an update has successfully processed.

Reply:
Removed it. Any case latest actually matters to the branch taken it's logged there, after being set to an actual time.